android/vcpkg_json_apps/vcpkg.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Please rename this folder to one of the next:
- apps
- vcpkg-config-apps
- vcpkg-apps

Other folders should be renamed correspondingly